include/alpaka/exec/IndependentElements.hpp

```diff
@@ -4,6 +4,7 @@
 #include "alpaka/idx/Accessors.hpp"
 
 #include <algorithm>
+#include <ciso646>
```

Comment:
I found that the inclusion of this header has no affect in C++ because C++ itself defines these operants.
@fwyzard Why do you added these header?

https://cplusplus.com/reference/ciso646/

Reply:
Because MSVC is a stupid compiler and _does not_ define the keywords by default: https://learn.microsoft.com/en-us/cpp/build/reference/microsoft-extensions-to-c-and-cpp?view=msvc-170&redirectedfrom=MSDN#iso646h-not-enabled .

Reply:
OK, MSVC _does_ define the keywords in c++20 mode: https://godbolt.org/z/eMre9PMvT .

So we can remove them again once we switch to c++20.

Reply:
Please add the comment `TODO: can be removed if C++ 20 is the minimum requirement`
This allows us to use `grep` for cleaning up the code, when we move to C++20.